src/Eppie.App/Eppie.App.ViewModels/SettingsPageViewModel.cs

```diff
@@ -91,6 +91,29 @@ public int ThemeSelectedIndex
             }
         }
 
+        public IReadOnlyList<AppScale> UiScaleOptions { get; } = new List<AppScale>
+        {
+            AppScale.SystemDefault,
+            AppScale.Scale100,
+            AppScale.Scale150,
+            AppScale.Scale200,
+            AppScale.Scale250,
+            AppScale.Scale300,
+        };
+
+        public AppScale SelectedUiScale
+        {
+            get => LocalSettingsService?.UiScale ?? AppScale.SystemDefault;
+            set
+            {
+                if (LocalSettingsService != null && LocalSettingsService.UiScale != value)
+                {
+                    LocalSettingsService.UiScale = value;
+                    OnPropertyChanged();
+                }
+            }
+        }
```

Comment:
Perhaps we should rename `SelectedUiScale` to 'SelectedUIScale'
Perhaps we should rename `UiScale ` to 'UIScale '